examples/integrations/cdi/oci-objectstorage/pom.xml

```diff
@@ -51,8 +51,8 @@
 
```

Comment:
Can you remove `<type>pom</type>` from above?

Reply:
At line 44?  I can.  That's been in there forever; it was originally how OCI wanted you to consume the two artifacts it aggregates.  Maybe they've changed that recommendation.

Reply:
Ah; I see now they've made it an [empty jar file that does the same thing](https://github.com/oracle/oci-java-sdk/tree/master/bmc-objectstorage/bmc-objectstorage-combined).  Let me dig in here; both of these situations seem like perhaps bad ideas.

Reply:
Looks like the empty jar project (`oci-objectstorage-combined`) has an `artifactId` of `oci-objectstorage` and a type of `jar`, and all it does is pull in transitive dependencies (`oci-objectstorage-extensions`, `oci-objectstorage-generated`).  Nevertheless it does indeed appear to be how OCI wants you to depend on Object Storage artifacts, so I'll remove the `<type>pom</type>` as suggested, and in `dependencies/pom.xml` as well.  Addressed in [75add80](https://github.com/oracle/helidon/pull/4029/commits/75add80219c55d498007c299e46da4680c478ba1).